container/root/scripts/install_s6.sh

```diff
@@ -14,13 +14,9 @@ curl -fL https://github.com/just-containers/s6-overlay/releases/download/${S6_VE
 curl https://keybase.io/justcontainers/key.asc| gpg --no-tty --batch --import
 gpg --no-tty --batch --verify /tmp/${S6_NAME}.sig /tmp/${S6_NAME}
 
-# Special handling - CentOS
+# Special handling - CentOS >= 7 + Ubuntu >= 20.04
 # @see https://github.com/just-containers/s6-overlay#bin-and-sbin-are-symlinks
-if [ -f /etc/centos-release ]; then
-  tar xzf /tmp/${S6_NAME} -C / --exclude="./bin"
-  tar xzf /tmp/${S6_NAME} -C /usr ./bin
-else
-  tar xzf /tmp/${S6_NAME} -C /
-fi
+tar xzf /tmp/${S6_NAME} -C / --exclude="./bin"
```

Comment:
symlinks for `bin/bash` did break shell without this fix. It is backwards compatible

Reply:
@bryanlatten Does this apply to alpine images as well?

Reply:
yes, it applies to all

Reply:
If we can guarantee that we're using GNU tar - we can combine these 2 tar commands with:

```
tar xzf /tmp/${S6_NAME} -C / --keep-directory-symlink
```

Reply:
I just checked BSD tar and it does not have `--keep-directory-symlink` so I think leaving it w/ 2 tar commands would be best.